src/bsp/Bsp.cpp

```diff
@@ -25,9 +26,21 @@ Bsp::Bsp() {
 	header.nVersion = 30;
 
 	for (int i = 0; i < HEADER_LUMPS; i++) {
-		header.lump[i].nLength = 0;
 		header.lump[i].nOffset = 0;
-		lumps[i] = NULL;
+		switch (i)
+		{
+		case LUMP_TEXTURES:
+			header.lump[i].nLength = 4;
+			memset(lumps[i], 0, header.lump[i].nLength);

```

Comment:
don't think the lumps have been allocated yet

Reply:
Yes sorry i edit via phone))

Reply:
Need fix)

Reply:
@wootguy now it fixed?

Reply:
yep, seeing `new` before `memset` now